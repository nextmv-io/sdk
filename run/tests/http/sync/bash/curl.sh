if false; then
go run main.go
fi
sleep 0.5
go run main.go > /dev/null 2>&1 &
sleep 3.5
PID2=$(lsof -i -P | grep LISTEN | grep :9000 | tr -s ' ' | cut -d ' ' -f 2)
curl -s -X POST "http://localhost:9000?duration=500000000" -H 'Content-Type: application/json' -d '{"message":"Hello"}' | jq
kill $PID2 > /dev/null 2>&1
exit 0
