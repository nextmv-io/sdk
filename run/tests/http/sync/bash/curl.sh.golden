{
  "version": {
    "sdk": "(devel)"
  },
  "options": {
    "duration": 500000000
  },
  "solutions": [
    {
      "message": "Hello World!"
    }
  ]
}
